app/models/CSVModels.js

```diff
@@ -0,0 +1,26 @@
+import csv from 'csvtojson';
```

Comment:
This file should be singular, named `Inspiration`, or whatever the actual model is named in Mongo.